src/secrets-kubernetes/Cargo.toml

```diff
@@ -13,3 +13,7 @@ mz-secrets = { path = "../secrets" }
 k8s-openapi = { version = "0.14.0", features = ["v1_22"] }
 kube = { version = "0.71.0", features = ["ws"] }
 tracing = "0.1.34"
+tokio = { version = "1.17.0" }
+rand = "0.8.5"
+async-std = "1.11.0"
```

Comment:
Ah, sorry, introducing a dependency on async-std to get access to an async `Path::exists` is a no go. async-std is a complete alternative to Tokio. We're a Tokio shop so we need to stick with crates in the Tokio ecosystem.

Unfortunately it looks like Tokio does not have an exact equivalent of `Path::exists`: https://github.com/tokio-rs/tokio/pull/3375. Long discussion there. But the upshot is there's a code snippet at the end of the issue that you can copy: https://github.com/tokio-rs/tokio/pull/3375#issuecomment-899107469

Reply:
I'm amazing this even works, `async-std` must be starting a runtime secretly or something